akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala

```diff
@@ -411,7 +411,7 @@ class CircuitBreaker(
       callTimeout)
 
   /**
-   * Java API for [[#withSyncCircuitBreaker]]. Throws [[java.util.concurrent.TimeoutException]] if the call timed out.
+   * Java API for [[withSyncCircuitBreaker[T](body:=>T):T* withSyncCircuitBreaker(=>T)]]. Throws [[java.util.concurrent.TimeoutException]] if the call timed out.
```

Comment:
Is the generated JavaDoc not rendering these links correctly as in your screen dump? Then I don't see why we would change this.

I suggest that we copy the doc comment from the other method here as well. A Java user is not interested in that there is another method for Scala.

Reply:
@patriknw Do you mean to copy documentation from [withSyncCircuitBreaker(body)](https://github.com/akka/akka/blob/031886a7b32530228f34176cd41bba9d344f43bd/akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala#L389)  

and from [withSyncCicuitBreaker(body,defineFailureFN](https://github.com/akka/akka/blob/031886a7b32530228f34176cd41bba9d344f43bd/akka-actor/src/main/scala/akka/pattern/CircuitBreaker.scala#L405) for that methods changed by me?

Reply:
@Captain1653, I think the point is that we don't need to tell what is the equivalent signature in Scala. 

It's enough to say `[[#withSyncCircuitBreaker]]`.
Like is done for the other methods. We just refer the method name and that's it.

But that's exactly the previous code was and then the PR won't be fixing anything. 😞

Reply:
@octonato Thank you for the explanation. I'll close my PR. It is not a problem. I have tried :) . Few  questions:
1) Is it worth to create separated issue in  [genjavadoc plugin](https://github.com/lightbend/genjavadoc) ? Looks like the plugin should generate correct javadoc from scaladoc.
2) Is it worth to add comment to    #19999  about it? I mean "hold the task until... future, because genjavadoc plugin"?